### ISSUE TITLE
Sanitize script output

### ISSUE THREAD

**Issue by mr-tz** [fireeye/flare-floss#116]
sanitize_string_for_script(s) in main()

**Comment by mr-tz** [comment]
Does encode('unicode_escape') work for this? See https://github.com/fireeye/flare-floss/blob/3c35cab2f0dfb6232ba7a0ad2f88e6607d1c5db6/floss/main.py#L69

```
Produce a string that is suitable as Unicode literal in Python source code.
- https://docs.python.org/2/library/codecs.html#standard-encodings
```

**Comment by williballenthin** [comment]
closed in #181
